powerhome/playbook
# [PLAY-490] Removed FA from Dist, Imported into Playbook via Yarn

#### Screens

## Icons rendering as expected
![Screen Shot 2022-11-16 at 1 02 11 PM](https://user-images.githubusercontent.com/73710701/202258245-baae2cf5-be58-4428-9443-1257786aaf49.png)

![Screen Shot 2022-11-16 at 1 02 38 PM](https://user-images.githubusercontent.com/73710701/202258311-67c46fac-7374-41c7-bd7a-bc7d20b42dbc.png)


#### Breaking Changes

YES. This PR removes Fontawesome from our dist and brings it in directly through yarn. Fonst awesome pro installed in playbook-website, fontawesome free installed in playbook

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-490)

#### How to test this

Test any kits that use icons, such as Icon, Icon Stat Value, Icon Circle, etc. Make sure all icons rendering

#### Checklist:
